app/src/main/res/layout/activity_bill.xml

```diff
@@ -18,323 +18,325 @@
 
-                            <android.support.v7.widget.CardView
+                            <TextView
                                 android:layout_width="wrap_content"
                                 android:layout_height="wrap_content"
-                                android:paddingEnd="@dimen/dp_10"
-                                android:paddingLeft="@dimen/dp_10"
-                                android:paddingRight="@dimen/dp_10"
-                                android:paddingStart="@dimen/dp_10"
-                                app:cardBackgroundColor="@color/colorPrimary"
-                                app:cardCornerRadius="@dimen/dp_5"
-                                car_view:cardElevation="@dimen/dp_2"
-                                >
-
-                                <ImageView
+                                android:layout_marginEnd="@dimen/dp_5"
+                                android:layout_marginLeft="@dimen/dp_5"
+                                android:layout_marginRight="@dimen/dp_5"
+                                android:layout_marginStart="@dimen/dp_5"
+                                android:textColor="@color/color_green_accent_400"
+                                android:textSize="@dimen/sp_10"
+                                android:textStyle="bold"/>
+
+                            <android.support.design.widget.TextInputLayout
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                bind:errorText="@{viewModel.formError}">
+
+                                <android.support.v7.widget.CardView
                                     android:layout_width="wrap_content"
                                     android:layout_height="wrap_content"
-                                    android:layout_marginTop="@dimen/dp_2"
-                                    android:onClick="@{()-> viewModel.onAddBillClick()}"
-                                    android:padding="@dimen/dp_3"
-                                    app:srcCompat="@drawable/ic_cart_60dp"
-                                    tools:ignore="ContentDescription"/>
-                            </android.support.v7.widget.CardView>
-                        </android.support.design.widget.TextInputLayout>
+                                    android:paddingEnd="@dimen/dp_10"
+                                    android:paddingLeft="@dimen/dp_10"
+                                    android:paddingRight="@dimen/dp_10"
+                                    android:paddingStart="@dimen/dp_10"
+                                    app:cardBackgroundColor="@color/colorPrimary"
+                                    app:cardCornerRadius="@dimen/dp_5"
+                                    car_view:cardElevation="@dimen/dp_2"
+                                    >
+
+                                    <ImageView
```

Comment:
WARNING: [TooDeepLayout] `activity_bill.xml` has more than 10 levels, bad for performance
 Layouts with too much nesting is bad for performance. Consider using a flatter layout (such as `RelativeLayout` or `GridLayout`).The default maximum depth is 10 but can be configured with the environment variable `ANDROID_LINT_MAX_DEPTH`.